packages/app/src/cli/services/dev/select-store.ts

```diff
@@ -39,12 +46,16 @@ export async function selectStore(
   org: Organization,
   developerPlatformClient: DeveloperPlatformClient,
 ): Promise<OrganizationStore> {
-  const showDomainOnPrompt = developerPlatformClient.clientName === ClientName.AppManagement
+  const useAppManageClient = developerPlatformClient.clientName === ClientName.AppManagement
   // If no stores, guide the developer through creating one
   // Then, with a store selected, make sure its transfer-disabled, prompting to convert if needed
-  let store = await selectStorePrompt(stores, showDomainOnPrompt)
+  let store = await selectStorePrompt(stores, useAppManageClient)
   if (!store) {
-    outputInfo(`\n${await CreateStoreLink(org.id)}`)
+    if (useAppManageClient) {
+      outputInfo(`\n${await CreateDevStoreLink(org.id)}`)
+    } else {
+      outputInfo(`\n${await CreateStoreLink(org.id)}`)
+    }
```

Comment:
I'm aware falls into the "sprinkling" issue @shauns mentioned a while back; wondering if folks have any other suggestions? Initial thought was to create another  `selectStore` which is used to manage stores from `AppManageClient`, but we inevitably still have to have this `if (useAppManageClient)` check for that logic.

This is also a bit clunkier than just checking in `CreateStoreLink`, but I chose to put this here b/c it's more straightforward and I want to avoid passing down parameters that will inevitably removed.